Signal/test/Models/MesssagesBubblesSizeCalculatorTest.swift

```diff
@@ -22,86 +17,94 @@ class FakeiPhone6JSQMessagesCollectionViewFlowLayout: JSQMessagesCollectionViewF
     }
 
     func testHeightForLong1LineMessage() {
         let text = "1 2 3 4 5 6 7 8 9 10 11 12 13 14 x"
-        let messageData = self.messageDataForForText(text)
-        let actual = calculator.messageBubbleSize(for: messageData, at: indexPath, with: layout)
-        XCTAssertEqual(38, actual.height)
+        let viewItem = self.viewItemForText(text)
+        let actual = messageBubbleSize(for: viewItem)
+        XCTAssertEqual(64, actual.height)
     }
 
     func testHeightForShort2LineMessage() {
         let text = "1 2 3 4 5 6 7 8 9 10 11 12 13 14 x 1"
-        let messageData = self.messageDataForForText(text)
-        let actual = calculator.messageBubbleSize(for: messageData, at: indexPath, with: layout)
-        XCTAssertEqual(59, actual.height)
+        let viewItem = self.viewItemForText(text)
+        let actual = messageBubbleSize(for: viewItem)
+        XCTAssertEqual(64, actual.height)
     }
 
     func testHeightForLong2LineMessage() {
         let text = "1 2 3 4 5 6 7 8 9 10 11 12 13 14 x 1 2 3 4 5 6 7 8 9 10 11 12 13 14 x"
-        let messageData = self.messageDataForForText(text)
-        let actual = calculator.messageBubbleSize(for: messageData, at: indexPath, with: layout)
-        XCTAssertEqual(59, actual.height)
+        let viewItem = self.viewItemForText(text)
+        let actual = messageBubbleSize(for: viewItem)
+        XCTAssertEqual(86, actual.height)
     }
 
     func testHeightForiOS10EmojiBug() {
-        let messageData = self.messageDataForForText("Wunderschönen Guten Morgaaaahhhn 😝 - hast du gut geschlafen ☺️😘")
-        let actual = calculator.messageBubbleSize(for: messageData, at: indexPath, with: layout)
+        let viewItem = self.viewItemForText("Wunderschönen Guten Morgaaaahhhn 😝 - hast du gut geschlafen ☺️😘")
+        let actual = messageBubbleSize(for: viewItem)
 
-        XCTAssertEqual(85.5, actual.height)
+        XCTAssertEqual(108, actual.height)
     }
 
     func testHeightForiOS10EmojiBug2() {
-        let messageData = self.messageDataForForText("Test test test test test test test test test test test test 😊❤️❤️")
-        let actual = calculator.messageBubbleSize(for: messageData, at: indexPath, with: layout)
+        let viewItem = self.viewItemForText("Test test test test test test test test test test test test 😊❤️❤️")
+        let actual = messageBubbleSize(for: viewItem)
 
-        XCTAssertEqual(62, actual.height)
+        XCTAssertEqual(86, actual.height)
     }
 
     func testHeightForChineseWithEmojiBug() {
-        let messageData = self.messageDataForForText("一二三四五六七八九十甲乙丙😝戊己庚辛壬圭咖啡牛奶餅乾水果蛋糕")
-        let actual = calculator.messageBubbleSize(for: messageData, at: indexPath, with: layout)
+        let viewItem = self.viewItemForText("一二三四五六七八九十甲乙丙😝戊己庚辛壬圭咖啡牛奶餅乾水果蛋糕")
+        let actual = messageBubbleSize(for: viewItem)
         // erroneously seeing 69 with the emoji fix in place.
-        XCTAssertEqual(85.5, actual.height)
+        XCTAssertEqual(86, actual.height)
```

Comment:
Do these tests need to be updated now that https://github.com/WhisperSystems/Signal-iOS/commit/418fef33586718eeba1414f702a2780470a44505 is merged?

Reply:
Yup, done.